java/values.yaml

```diff
@@ -24,6 +24,10 @@ livenessFailureThreshold: 3
 devApplicationInsightsInstrumentKey: '00000000-0000-0000-0000-000000000000'
 useInterpodAntiAffinity: false
 ingressClass: traefik
+# WARNING: idamIngressSessionAffinity is a temporary option only approved for SIDAM
+# Do not use without approval from RPE
```

Comment:
This should read: "This is subject to removal without notice. Do NOT use for any reason!"

Reply:
```suggestion
# This is subject to removal without notice. Do NOT use for any reason!
```